packages/typespec-vscode/package.json

```diff
@@ -115,8 +115,16 @@
     "menus": {
       "explorer/context": [
         {
-          "command": "typespec.importFromOpenApi3",
-          "group": "typespec@1"
```

Comment:
@iscai-msft Do you have context why this import is deleted? It seems influence tpyespec-vscode.
@RodgeFu for awareness.